src/Components/OATGraphViewer/OATGraphViewer.styles.tsx

```diff
@@ -3,17 +3,95 @@ import {
     mergeStyleSets,
     useTheme,
     FontSizes,
-    IButtonStyles,
     IStyleFunctionOrObject,
     ILabelStyleProps,
     ILabelStyles,
-    IStackProps,
     IButtonProps,
     IIconStyles,
-    ITextFieldStyles
+    FontWeights
 } from '@fluentui/react';
-import { CSSProperties } from 'react';
 import { CardboardClassNamePrefix } from '../../Models/Constants';
+import {
+    CONTROLS_BOTTOM_OFFSET,
+    CONTROLS_CALLOUT_OFFSET,
+    CONTROLS_SIDE_OFFSET,
+    CONTROLS_Z_INDEX,
+    getControlBackgroundColor,
+    LOADING_Z_INDEX,
+    PANEL_VERTICAL_SPACING,
+    PROPERTY_EDITOR_WIDTH
+} from '../../Models/Constants/OatStyleConstants';
+import { HEADER_BUTTON_HEIGHT } from '../../Models/Constants/StyleConstants';
+import { useExtendedTheme } from '../../Models/Hooks/useExtendedTheme';
+import {
+    IOATGraphViewerStyleProps,
+    IOATGraphViewerStyles
+} from './OATGraphViewer.types';
+
+export const classPrefix2 = `${CardboardClassNamePrefix}-oat-graph-viewer`;
```

Comment:
The value for `classPrefix2` is the same as `classPrefix`. Why not reuse and merge `classNames` and `classNames2`?

Reply:
It's a fair point. I was trying to convert the new parts over to a new set of styles and break off from the giant mess they had in here already that uses the old styling pattern. It just made it easier to know which ones I created and which ones are legacy that still need to get cleaned up. There's likely some in there that aren't used at all but I wanted to do a later pass to purge outdated styles.